dev-tools/mage/copy.go

```diff
@@ -104,20 +104,20 @@ func (t *CopyTask) fileCopy(src, dest string, info os.FileInfo) error {
 	return destFile.Close()
 }
 
-func (t *CopyTask) dirCopy(src, dest string, info os.FileInfo) error {
+func (t *CopyTask) dirCopy(src, dest string, entry fs.DirEntry) error {
 	if t.isExcluded(src) {
 		return nil
 	}
 
 	mode := t.DirMode
 	if mode == 0 {
-		mode = info.Mode()
+		mode = entry.Type()
 	}
 	if err := os.MkdirAll(dest, mode&os.ModePerm); err != nil {
 		return errors.Wrap(err, "failed creating dirs")
 	}
 
-	contents, err := ioutil.ReadDir(src)
+	contents, err := os.ReadDir(src)
```

Comment:
Conversion to os.ReadDir, the return type changed to fs.DirEntry. Double check this code.